examples/environment-variables/src/verify_command.sh

```diff
@@ -4,6 +4,6 @@ echo "# you can edit it freely and regenerate (it will not be overwritten)"
 inspect_args
 
 echo "environment:"
-echo "- API_KEY=$API_KEY"
+echo "- API_KEY=${API_KEY:-}"
```

Comment:
As you can see I also had to make some changes to some of the examples to make them safe as well.

Reply:
This is exactly one of the reasons I am not a fan of strict setup.
In some cases, it is perfectly legitimate to have empty, unset variables - it is a part of nature... :)

Reply:
Yes, as a rule I do not want to force this on users. But it was useful to make all of the examples work in this mode, so that I could run the tests in this mode and easily make sure that all of the different ways of generating a script were still working.